train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -7,6 +7,20 @@
 	private int step = 0;
 	private int referenceSpeed = 0;
 	private int speedLimit = 0;
+	private Thread thread;
+
+	public TrainControllerImpl() {
+		thread = new Thread() {
+			public void run() {
+				thread.run();
+				try {
+					followSpeed();
+					thread.sleep(1000);
```

Comment:
We said this should be 1250 but after i consulted with Siemens Engineers we decided to leave it 1000